### PR TITLE
Shopper → Mini Cart → Can go to Cart page

### LINKED ISSUE #5836: Critical flows: Shopper → Mini Cart → Can go to Cart page
### Note

Create e2e tests to verify that a shopper can go to the Cart page from the Mini Cart drawer.

### File

`tests/e2e/specs/shopper/mini-cart.test.js`

Part of #5814

### PR DESCRIPTION
**Pull request by dinhtungdu**
Fixes #5836

### Notes

This PR adds e2e tests to verify that a shopper can go to the Cart page from the Mini Cart drawer.

#### Automated Tests
* [x] Changes in this PR are covered by Automated Tests.
  * [ ] Unit tests
  * [x] E2E tests

### Manual Testing

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e -- tests/e2e/specs/shopper/mini-cart.test.js -t "Cart page"` to run only this test case.
5. Run `npm run test:e2e` to run the whole test suite.

### CODE REVIEW

File: tests/e2e/specs/shopper/mini-cart.test.js [hunk 1]
```diff
@@ -218,4 +219,41 @@ describe( 'Shopper → Mini Cart', () => {
...
 		} );
 	} );
+
+	describe( 'Cart page', () => {
+		beforeAll( async () => {
+			await shopper.emptyCart();
+		} );
+
+		it( 'Can go to cart page from the Mini Cart Footer', async () => {
+			const productTitle = await page.$eval(
```
Comment (gigitux) [inline tests/e2e/specs/shopper/mini-cart.test.js]: Just checking what does `$$eval`, I found this utility in our codebase, `getTextContent`: https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/f79618fe252ebe4ed97f6059e6fd81c149dd64aa/tests/e2e/page-utils.js#L53
I guess that we should use it

File: tests/e2e/specs/shopper/mini-cart.test.js [hunk 2]
```diff
@@ -218,4 +219,41 @@ describe( 'Shopper → Mini Cart', () => {
...
+			);
+
+			const cartUrl = await page.$eval(
+				'.wc-block-mini-cart__footer-cart',
+				( el ) => el.href
+			);
+
+			expect( cartUrl ).toMatch( SHOP_CART_PAGE );
+
+			await page.goto( cartUrl, { waitUntil: 'networkidle0' } );
```
Comment (gigitux) [inline tests/e2e/specs/shopper/mini-cart.test.js:252]: Is there a specific reason we don't simulate the click on the button? I guess that we should test it. Maybe we could add  `e.preventDefault()` that prevents the default behavior in the future.

**Review by gigitux — CHANGES_REQUESTED** [review]
Overall looks great!
I added two comments, but I'm not very expert with E2E tests. Let me know what you think

If you can, it would be great if you can resolve the conflict too 💪

File: tests/e2e/specs/shopper/mini-cart.test.js (hunk 2 above)
Comment (dinhtungdu) [reply tests/e2e/specs/shopper/mini-cart.test.js:252]: Clicking directly on a link may cause racing conditions leads to flaky issue sometime. I use goto for more realiable.

File: tests/e2e/specs/shopper/mini-cart.test.js (hunk 1 above)
Comment (dinhtungdu) [reply tests/e2e/specs/shopper/mini-cart.test.js]: Yes. Thanks for the head-up! I'm already using it in #5979 but forgot to update this one!

**Review by gigitux — APPROVED** [review]
Thanks for update the PR! LGTM! 🥳

**Pull request merged**